parlai/core/torch_generator_agent.py

```diff
@@ -354,6 +354,13 @@ def upgrade_opt(cls, opt_from_disk: Opt):
                 ]
             del opt_from_disk['beam_blacklist_filename']
 
+        # 2020-08-04: Introduce full context beam blocking
+        # Previous, specifying --beam-context-block-ngram > 1 would block
+        # from generating ngrams from model's context, which is limited
+        # by truncation parameters. Now, we block on full dialogue history.
+        if 'beam_block_full_context' not in opt_from_disk:
+            opt_from_disk['beam_block_full_context'] = False
```

Comment:
Hmm should we perhaps print a message about this so that people know?

Reply:
ok, added